### PR TITLE
ER-1504: Install orlaprisen overlay

### PR DESCRIPTION
**Pull request by yepzdk**
#### Link to ticket

https://jira.itkdev.dk/browse/ER-1504

#### Description

Implement overlay from HelloMondy

#### Screenshot of the result

Screenshot 2023-08-25 at 11 28 38

#### Checklist

- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: sites/all/themes/wille/templates/system/html.tpl.php [hunk 1]
```diff
@@ -59,5 +59,6 @@
   <?php print $page; ?>
   <?php print $scripts; ?>
   <?php print $page_bottom; ?>
+  <script src="/sites/all/themes/wille/overlay/overlay.js"></script>
```
Comment (rimi-itk) [inline sites/all/themes/wille/templates/system/html.tpl.php:62]: Use https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_get_path/7.x to get the theme path.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

**Review by rimi-itk — APPROVED** [review]

File: sites/all/themes/wille/overlay/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Note about video paths
+
+I have changed the paths to the videos in  `overlay.js` in order for them to load.
+
+```js
+const chosenVideo = [
+      {
+        videoPathMov: "sites/all/themes/wille/overlay/videos/orla/orla.mov",
+        videoPathWebm: "sites/all/themes/wille/overlay/videos/orla/orla.webm",
```
Comment (rimi-itk) [inline sites/all/themes/wille/overlay/README.md:21]: Remove the `mp4` files if they are not used.

File: sites/all/themes/wille/templates/system/html.tpl.php [hunk 3]
```diff
@@ -59,5 +59,6 @@
   <?php print $page; ?>
   <?php print $scripts; ?>
   <?php print $page_bottom; ?>
+  <script src="<?php print drupal_get_path('theme', 'wille') . '/overlay/overlay.js?2'; ?>"></script>
```
Comment (rimi-itk) [inline sites/all/themes/wille/templates/system/html.tpl.php:62]: Do they really want this on all pages? If not, we have to find a different way to insert the `script` element.

File: sites/all/themes/wille/overlay/README.md
Comment (rimi-itk) [inline sites/all/themes/wille/overlay/README.md:1]: This looks more like (internal) developer documentation than deployment documentation. All the changes mentioned have already been made in the code, so we don't really need to tell about them here. I think we should remove the `README.md` file.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: sites/all/themes/wille/templates/system/html.tpl.php (hunk 3 above)
Comment (yepzdk) [reply sites/all/themes/wille/templates/system/html.tpl.php:62]: They would like to test it only on the frontpage. Do you have any suggestions on how to do that?

Comment (rimi-itk) [reply sites/all/themes/wille/templates/system/html.tpl.php]: @yepzdk, https://www.drupal.org/forum/support/module-development-and-code-questions/2011-10-01/adding-javascript-to-front-page-only#comment-5064274 may be the way to do it. Or at least worth a try.

Comment (yepzdk) [reply sites/all/themes/wille/templates/system/html.tpl.php]: Thanks @rimi-itk

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**